compiler/passes/checkResolved.cpp

```diff
@@ -125,6 +125,10 @@ isDefinedAllPaths(Expr* expr, Symbol* ret, RefSet& refs)
     if (call->isPrimitive(PRIM_RT_ERROR))
       return 1;
 
+    if (call->isPrimitive(PRIM_THROW)) {
```

Comment:
Could you update the comment above

> // Returns the smallest number of definitions of ret on any path through the
> // given expression.

to indicate explicitly how return/throw is handled and why this is right?